ci-scripts/compile-settings.jq

```diff
@@ -1,7 +1,9 @@
 if $for_release then {
   output: "release",
   profile: "release",
-  args: ($matrix.release_build_args // ""),
+  # Use build-std to build a std library optimized for size and abort immediately on abort,
+  # so that format string for `unwrap`/`expect`/`unreachable`/`panic` can be optimized out.
+  args: ($matrix.release_build_args // "-Z build-std=std,panic_abort -Z build-std-features=panic_immediate_abort"),
```

Comment:
do you want to always have these args? as is windows won't do the build-std thing

Reply:
Actually it works on windows , but failed on armv7, aarch64 musl build, due to missing headers in cross.

Reply:
@passcod https://github.com/NobodyXu/cargo-binstall/actions/runs/2759609371